product_docs/docs/edbcloud/beta/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Understanding resource requirements"
+
+---
+
+By default, Azure sets low resource limits on the number of vCPUs (cores), and Public IP addresses permitted in your subscription. The following sections explain BigAnimal's requirements and recommended resource limits in Azure.
+ 
+### Public IP addresses limits
```

Comment:
This should describe how and why BigAnimal uses public IP addresses.

Reply:
Added this line: 
"A Public IP address is an IP address that used to access your cluster directly over the internet. The number of Public IP addresses must not exceed the limit for a particular region."